README.md

```diff
@@ -1 +1,45 @@
-# moneymaker
\ No newline at end of file
+# moneymaker
+
+budget calculator
+1. user input (hardcoded)
+    1. String date
+    1. String description
+    1. int amount
+    1. int outgoing
+1. display to user
+    1. int balance projection
+
+Classes in use:
+1. Moneymaker (main method)
+    - creates a new Budget using:
+    - new Budget(new Balance(), new TransactionMaker().getTransactions()).printReport();
+    Todo: is TransactionMaker really the most simple way to get data into the system?
+1. Budget:
+    - Budget(Balance balance, List<Transaction> transactions)
+    - updateBalance()
+    - printBalance()
+1. Balance:
+    - get()
+    - increment(int i)
+1. TransactionMaker
+    - Hardcode the transactions with makeTransaction()
+    - Uses the Transaction object
+1. Transaction: (a POJO)
+     - String time
+     - TransactionType transactionType: (enum CREDIT/DEBIT)
+     - int amount
+     - String description
+     - String reportStringFormat()
+     Todo: should reportStringFormat() be in it's own class?
+
+transaction.reportStringFormat() should look like:
+UTC, transactionType, value, description, balance after transaction.
+
+TODO: unable to print balance after each transaction - I can only print the final balance.
```

Comment:
I think we can figure out where we should put this if we do some debugging.
Alternatively, we can do this in a very clean manner using TDD as well.